include/logger.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef OBC_LOGGER_HPP
+#define OBC_LOGGER_HPP
+
+#include <Arduino.h>
+#include <SD.h>
+#include <Wire.h>
+
+#include "acceleration.hpp"
+#include "barometer.hpp"
+#include "error.hpp"
+#include "gps.hpp"
+
+namespace obc {
+constexpr uint8_t cs = 4;
```

Comment:
co to?

Reply:
faktycznie cs - chip_select (od karty sd) to niejasny skrót, poprawiłem nazwę i wrzuciłem to do pliku `logger.cpp`